src/main/scala/org/phenoscape/kb/Similarity.scala

```diff
@@ -233,6 +237,53 @@ object Similarity {
       } yield JaccardScore(Set(left, right), intersectionCount.toDouble / unionCount.toDouble)).toSeq
     }
 
+  // Output a boolean matrix as CSV
+  def matrixRendererFromMapOfMaps(mapOfMaps: Map[Any, Map[Any, Boolean]]) = {
+
+    val sortedKeys = mapOfMaps.keys.toList.map(_.toString).sorted
+    var resultString = "headers, " + sortedKeys.mkString(", ") //print column headers
+
+    for(x <- sortedKeys) yield {
```

Comment:
Put a space between `for` and what follows. This will probably happen if you do auto-format with IntelliJ.